scenario/LiquidationBotScenario.ts

```diff
@@ -604,7 +627,7 @@ scenario(
     upgrade: {
       targetReserves: exp(20_000, 18)
     },
-    filter: async (ctx) => matchesDeployment(ctx, [{deployment: 'mainnet'}]), // XXX enable for Polygon
+    filter: async (ctx) => matchesDeployment(ctx, [{network: 'mainnet'}]), // XXX enable for Polygon
```

Comment:
When should we enable this?

Reply:
Good call out. Don't see a reason not to. WDYT🥈?

Reply:
There's nothing, standing in our way to enable this for Polygon. I just suspect that it's going to be even more flakey than the other Liquidation Bot scenarios (which are already pretty flakey).